--- build.rs.orig	2023-08-21 17:51:24.722613500 +0900
+++ build.rs	2023-08-21 17:59:24.117969800 +0900
@@ -101,7 +101,8 @@
     println!("cargo:rustc-link-lib=static=JavaScriptCore");
     println!("cargo:rustc-link-lib=static=WTF");
     println!("cargo:rustc-link-lib=static=bmalloc");
-
+    println!("cargo:rustc-link-lib=atomic");
+    
     if cfg!(target_os = "macos") {
         // x86_64-apple-darwin
         println!("cargo:rustc-link-lib=icucore");
